bin/posydon-run-pipeline

```diff
@@ -701,9 +701,8 @@ def copy_ini_file(grid_type, rerun_metallicity, rerun_type, destination,
 #        replace_text = "matthias_LBV_wind-051343856f50dc5518abdf28904d62df4caacaff" #TODO: update to new MESA-INLIST branch/commit
     elif rerun_type == 'LBV_wind+dedt_energy_eqn':
         replace_text = "seth_lbv_thermo_dedt-936f76f82d44b16139ff89e41e5b0f2232fd9b30"
-#    elif rerun_type == 'LBV_wind+dedt_hepulse':
-#        raise ValueError(f'Not yet supported rerun type {rerun_type}!')
-#        replace_text = "matthias_LBV_wind-051343856f50dc5518abdf28904d62df4caacaff" #TODO: update to new MESA-INLIST branch/commit
+    elif rerun_type == 'LBV_wind+dedt_hepulse':
+        replace_text = "seth_lbv_thermo_dedt_hepulse-df8c7a1fbf35b2de06e3c0b2a913d9be012e54ee" #TODO: update to new MESA-INLIST branch/commit
```

Comment:
I think we need a different branch here, which included the noAgeLimit changes, otherwise we can't apply it on the CO-HMS grid.